nav2_recoveries/src/recovery_server.cpp

```diff
@@ -73,21 +73,19 @@ RecoveryServer::on_configure(const rclcpp_lifecycle::State & /*state*/)
   tf_->setCreateTimerInterface(timer_interface);
   transform_listener_ = std::make_shared<tf2_ros::TransformListener>(*tf_);
 
-  std::string costmap_topic, footprint_topic;
+  std::string costmap_topic, footprint_topic, robot_base_frame;
+  double transform_timeout;
   this->get_parameter("costmap_topic", costmap_topic);
   this->get_parameter("footprint_topic", footprint_topic);
-  this->get_parameter("transform_tolerance", transform_tolerance_);
+  this->get_parameter("transform_timeout", transform_timeout);
```

Comment:
transform_tolerance is the standardized parameter name for transform timeout across all of the packages, please change that name back